auth/src/main/java/org/aerogear/mobile/security/checks/ScreenLockCheck.java

```diff
@@ -20,8 +20,9 @@
     /**
      * Check whether the device has a screen lock enabled (PIN, Password, etc).
      *
-     * @param context Context to be used by the check.
-     * @return <code>true</code> if the device has a screen lock enabled.
+     * @param context {@link Context} to be used by the check
+     * @return <code>true</code> if the device has a screen lock enabled
+     * @throws IllegalArgumentException if context is null
```

Comment:
IllegalArgumentException is not thrown here. Context is guaranteed to be non null by the parent class.